pkgs/applications/office/paper-note/default.nix

```
@@ -44,6 +44,10 @@ stdenv.mkDerivation rec {
     gtksourceview5
   ];
 
+  patchPhase = ''
+    sed -i 's/1.2.0/1.2.1/g' src/meson.build
```

```suggestion
    sed -i 's/1.2.0/${libadwaita.version}/g' src/meson.build
```

Also consider using `substituteInPlace`.